annotation/tests/test_tasks_services.py

```diff
@@ -1746,3 +1746,120 @@ def test_get_tasks_without_ids_with_multiple_task_types():
         4: [({"description": "Task 6"}, "6")],
     }
     assert services.get_tasks_without_ids(tasks) == expected
+
+
+def test_create_validation_revisions_successful(
+    mock_session: Mock,
+    mock_get_file_path_and_bucket: Mock,
+    mock_construct_annotated_pages: Mock,
+    mock_construct_annotated_doc: Mock,
+    mock_update_task_status: Mock,
+    mock_logger_exception: Mock,
```

Comment:
fix typings, please. not all of your fixtures return mocks

Reply:
This issue is from before the test changes were made.